### PR TITLE
DIV-7904 | Revoke certificate

### PR DESCRIPTION
**Pull request by varadeth**
Revoke previously generated certificate with preEnrollmentCode and dose as request parameters

### CODE REVIEW

File: backend/vaccination_api/config/config.go [hunk 1]
```diff
@@ -32,7 +33,7 @@ func updatePublicKeyFromKeycloak() error {
 			Config.Keycloak.Pubkey = publicKey
 		}
 	}
-	return errors.New("Unable to get public key from keycloak")
+	return errors.New("unable to get public key from keycloak")
```
Comment (dileepbapat) [inline backend/vaccination_api/config/config.go:36]: Is this intentional? error message is getting concatenated ?

File: backend/vaccination_api/pkg/handler.go [hunk 2]
```diff
@@ -30,6 +28,7 @@ import (
 	"github.com/divoc/kernel_library/services"
 	"github.com/go-openapi/runtime"
 	"github.com/go-openapi/runtime/middleware"
+	"github.com/jinzhu/gorm"
```
Comment (dileepbapat) [inline backend/vaccination_api/pkg/handler.go:31]: are we using gorm here?

File: backend/vaccination_api/pkg/handler.go [hunk 3]
```diff
@@ -749,6 +749,127 @@ func postCertificateRevoked(params certificate_revoked.CertificateRevokedParams)
...
+								certificateFailedToAddToRevocationList = append(certificateFailedToAddToRevocationList, certificateId)
+							}
+						}
+						programId := body["programId"].(string)
+						key := preEnrollmentCode + "-" + programId + "-" + strconv.Itoa(int((certificateDose["dose"]).(float64)))
+						if config.Config.Redis.ProgramIdCaching == "false" {
+							key = preEnrollmentCode + "-" + strconv.Itoa(int((certificateDose["dose"]).(float64)))
+						}
+						log.Infof("Key for Redis : %v, %v", key, config.Config.Redis.ProgramIdCaching)
+						err := kafkaService.DeleteValue(key)
```
Comment (dileepbapat) [inline backend/vaccination_api/pkg/handler.go:802]: Should service name be redis?

File: backend/vaccination_api/pkg/handler.go (hunk 2 above)
Comment (varadeth) [reply backend/vaccination_api/pkg/handler.go:31]: yes dileep. we are using it
This change came in due to some VS Code plugin which changed the line nos for gorm import

File: backend/vaccination_api/config/config.go (hunk 1 above)
Comment (varadeth) [reply backend/vaccination_api/config/config.go:36]: There was warning here that error strings should not be capitalized

**Pull request closed without merging**